.bumpversion.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 3.12.0
+current_version = 3.13.0
```

Comment:
I just checked master against develop, and we never released yet 3.12, so no need to increase the version number, otherwise we are going to skip v3.12 in PyPI.